realtime/src/services/exchanges/currencybeacon/index.ts

```diff
@@ -0,0 +1,95 @@
+import axios from "axios"
+
+import {
+  InvalidTickerError,
+  ExchangeServiceError,
+  UnknownExchangeServiceError,
+  InvalidExchangeConfigError,
+} from "@domain/exchanges"
+import { toPrice, toSeconds, toTimestamp } from "@domain/primitives"
+import { LocalCacheService } from "@services/cache"
+import { CacheKeys } from "@domain/cache"
+import { baseLogger } from "@services/logger"
+
+export const CurrencyBeaconExchangeService = async ({
+  base,
+  quote,
+  config,
+}: CurrencyBeaconExchangeServiceArgs): Promise<
+  IExchangeService | ExchangeServiceError
+> => {
+  if (!config || !config.apiKey) return new InvalidExchangeConfigError()
+
+  const { baseUrl, apiKey, timeout, cacheSeconds, ...params } = config
+
+  const url = baseUrl || "https://api.currencybeacon.com/v1"
+  const cacheKey = `${CacheKeys.CurrentTicker}:currencybeacon:${base}:*`
+
+  const getCachedRates = async (): Promise<CurrencyBeaconRates | null> => {
```

Comment:
Why `null` and not `undefined`?

Reply:
fixed https://github.com/GaloyMoney/price/pull/36/commits/cfdd36a3a9a658db89bad689231b833480a3a69d